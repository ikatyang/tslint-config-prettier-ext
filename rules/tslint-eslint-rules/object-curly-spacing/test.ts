const x = {a:1};
