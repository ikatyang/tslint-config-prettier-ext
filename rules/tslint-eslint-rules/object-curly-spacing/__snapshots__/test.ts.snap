// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be pretty after formatting 1`] = `
"
<<<<<< before
const x = {a:1};

======
const x = { a: 1 };

>>>>>> after
"
`;

exports[`should fail before formatting 1`] = `
"ERROR: (object-curly-spacing) /rules/tslint-eslint-rules/object-curly-spacing/test.ts[1, 11]: A space is required after '{'
ERROR: (object-curly-spacing) /rules/tslint-eslint-rules/object-curly-spacing/test.ts[1, 15]: A space is required before '}'
"
`;
